SpatialGDK/Documentation/content/toolbar.md

```diff
@@ -25,7 +29,7 @@ Or inside the Unreal Editor, navigate to **Edit** > **Plugins** and scroll down
 
 ## Buttons
 
-The SpatialOS Unreal GDK toolbar has five features mapped to individual buttons, and is displayed in the main editor toolbar to the right of the `Launch` button:
+The SpatialOS Unreal GDK toolbar has five features mapped to individual buttons and is displayed in the main editor toolbar to the right of the `Launch` button:
```

Comment:
Could we refer to it as "the GDK toolbar" everywhere in the doc except for the first instance?